WalletWasabi.Fluent/Views/MainView.axaml

```diff
@@ -110,7 +111,14 @@
               IsEnabled="{Binding $parent.DataContext.IsDialogScreenEnabled}"
               HorizontalAlignment="Stretch" VerticalAlignment="Stretch"
               HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch"
-              MaxContentWidth="800" MaxContentHeight="700">
+              MaxContentWidth="800" MaxContentHeight="700"
+              IncreasedWidthThreshold="740" IncreasedHeightThreshold="580" FullScreenHeightThreshold="580">
+      <c:Dialog.Styles>
+        <Style Selector="c|Dialog[FullScreenEnabled=True]">
+          <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+          <Setter Property="VerticalContentAlignment" Value="Stretch" />
+        </Style>
+      </c:Dialog.Styles>
```

Comment:
Why was this needed?

Reply:
To fill entire area in full screen mode.

Reply:
L113 - Stretch is the default.

I didn't notice any difference without this code.

Reply:
Hm maybe something changed, anyway this was based on compact dialog screen style.

Reply:
At compact dialog, there is no set for VerticalContentAlignment and HorizontalContentAlignment so it uses the default which is the `Center` (set in dialog.axaml), and that is why it was needed to set it to `Stretch` when full screen triggered.

At normal dialog, they are set to Stretch so you can remove this whole style.

Reply:
ok will remove style

Reply:
Fixed